public/styles/styles.css

```diff
@@ -0,0 +1,19 @@
+body {
+  background: black;
+  text-align: center;
+  color: white;
+}
+
+header{
+  height: 200px;
+}
+
+main{
+  margin: 0 auto;
+  color: black;
+}
+footer {
+  position:fixed;
```

Comment:
Rather than keeping the footer fixed to the bottom of the window, you might make it absolute at the bottom of the page.